tests/test_suite_test.go

```diff
@@ -79,4 +81,6 @@ var _ = BeforeSuite(func() {
 	if runtimeClassName = os.Getenv(runtimeClassNameVarName); runtimeClassName == "" {
 		runtimeClassName = defaultRuntimeClassName
 	}
+
+	trafficGeneratorImage = os.Getenv(trafficGeneratorImageVarName)
```

Comment:
Why is this environment variable is treated differently than the others?

Reply:
I don't think it should have a default value set here, because we already do that on the config package...